app/views/About.js

```diff
@@ -125,38 +121,11 @@ export const AboutScreen = ({ navigation }) => {
 const styles = StyleSheet.create({
   contentContainer: {
     flexDirection: 'column',
-    width: '100%',
     backgroundColor: Colors.primaryBackground,
-    paddingHorizontal: 40,
-    paddingBottom: 42,
-  },
-  aboutLabelContainer: {
-    flexDirection: 'column',
-  },
-  aboutSectionIconLock: {
-    width: 20,
-    height: 26.67,
-    marginTop: 36,
-  },
-  aboutSectionTitles: {
-    color: Colors.primaryText,
-    marginTop: 14,
-    lineHeight: 32,
-  },
-  aboutSectionPara: {
-    color: Colors.primaryText,
-    fontSize: 16,
-    lineHeight: 22.5,
-    marginTop: 12,
-    fontFamily: fontFamily.primaryRegular,
+    paddingHorizontal: 24,
```

Comment:
```suggestion
    paddingHorizontal: Spacing.medium,
```

Minor nitpick, but mostly to encourage future contributors to staying within the design system!

Reply:
Good call, I've been lazy on these